charts/jiralert/templates/secret.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ include "jiralert.fullname" . }}
+  namespace: {{ template "jiralert.namespace" . }}
+  labels:
+    {{- include "jiralert.labels" . | nindent 4 }}
+data:
+  jiralert.yml: {{ tpl (.Values.config | toYaml) . | b64enc }}
+  jiralert.tmpl: {{ tpl .Values.issueTemplate . | b64enc }}
```

Comment:
it should also be possible to get these values from exisitng secrets.

Reply:
Done